lua/bufferline.lua

```diff
@@ -51,13 +52,17 @@ function bufferline.setup(user_config)
     function(tbl)
       local index = tonumber(tbl.args)
       if not index then
-        return notify('Invalid argument to `:BufferGoto`', vim.log.levels.ERROR)
+        return notify(
+          'Invalid argument to ' .. utils.markdown_inline_code':BufferGoto',
```

Comment:
I created a `markdown_inline_code` function to ensure that all <code>`</code> symbols are evenly wrapped.